cttportfolio.html

```diff
@@ -0,0 +1,106 @@
+<!doctype html>
+<html lang= "en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="UTF-8">
+	<title>HTML Exercise 1</title>
+	<link rel="stylesheet" type="text/css" href="cttportfolio.css">
+</head>
+<body>
+	<nav>
+		<a href="cttportfolio.html"><img src="https://placeholder.com/wp-content/uploads/2018/10/placeholder.com-logo3.png" alt="Logo replacement"></a> 
+		<div class="navlinks">
+			<a href="cttportfolio.html">Home</a>
+			<a href="google.com">About me</a>
+			<a href="google.com">Portolio</a>
+			<a href="google.com">Contact me</a>
+		</div>
+	</nav>
+	<main>
+		<header>
+			<div class="headertext">
+				<h1>Page Intro Text Here</h1>
+				<h2>Page intro subtext here</h2>	
+			</div>
+		</header>
+		<div class="aboutmediv">
+			<div class="aboutmemain">
+				<div class="aboutmemainchild">
+					<div class="aboutmeimg">
+						<img src="https://images.unsplash.com/photo-1507003211169-0a1dd7228f2d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80" alt="picture of me here">
+					</div>
+					<div class="aboutmetext">
+						<h4>about me</h4>
+						<p>About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here </p>
+					</div>
+				</div>
+				<button>button to expand subpage</button>
```

Comment:
You should give the `<button>` a `type` of `button`, I know it seems a bit weird but the default is `submit`. 🤷‍♂ 
```suggestion
				<button type="button">button to expand subpage</button>
```

Reply:
cool didn't know that!